test/integration/docs.test.js

```diff
@@ -17,7 +16,6 @@ describe('async-docs', function () {
   })
 
   it('successfully returns the asyncapi docs json', function () {
-    expect(context.response.body).to.be.a.jsonObj()
```

Comment:
the only place where `chai-json` was used

Reply:
can we have an alternate assertion?

Reply:
how about https://github.com/digicatapult/wasp-ws-reading-service/pull/2/commits/9fc8844a51b237c7a69df08d3be35b7fc51d56e1?

Reply:
ugly but it will do